### PR TITLE
Add Copy Screenshot to Clipboard on CTRL + Screenshot button

### PR DESCRIPTION
**Pull request by Zageron**
- Finish playing game.
- Press `CTRL + C`
- Paste in discord.

Requirements:
- Add language string `copy_image_to_clipboard`
- Add language string `game_results_queue_save_screenshot_clipboard_hint` to include tip about the CTRL key, leave old string as to not break old engines.

### CODE REVIEW

File: src/game/GameResults.as [hunk 1]
```diff
@@ -134,6 +134,7 @@ package game
         {
             // Add keyboard navigation
             stage.addEventListener(KeyboardEvent.KEY_DOWN, eventHandler);
+            stage.addEventListener(KeyboardEvent.KEY_UP, eventHandler);
```
Comment (Zageron) [inline src/game/GameResults.as:137]: @Psycast Do I need to add a remove event listener to the `else if (keyCode == _gvars.playerUser.keyQuit)` section which has one for KEY_DOWN?

Comment (Zageron) [reply src/game/GameResults.as]: Irrelevant based on comments in discord.

File: src/game/GameResults.as [hunk 2]
```diff
@@ -716,12 +717,22 @@ package game
 
             else if (target == navScreenShot)
             {
-                var ext:String = "";
-                if (resultIndex >= 0)
+                if (e.ctrlKey)
+                {
+                    navScreenShot.purge_hover_sprite();
```
Comment (Zageron) [inline src/game/GameResults.as:722]:
```suggestion
                navScreenShot.purge_hover_sprite();
                if (e.ctrlKey)
                {
```

File: src/game/GameResults.as [hunk 3]
```diff
@@ -716,12 +717,22 @@ package game
...
+                }
+                else
                 {
-                    ext = songResults[resultIndex].screenshot_path;
+                    var ext:String = "";
+                    if (resultIndex >= 0)
+                    {
+                        ext = songResults[resultIndex].screenshot_path;
+                    }
+                    navScreenShot.purge_hover_sprite();
```
Comment (Zageron) [inline src/game/GameResults.as:732]:
```suggestion
```

**Review by Psycast — DISMISSED** [review]

**Pull request merged**